.circleci/working_config.yaml

```diff
@@ -0,0 +1,78 @@
+version: 2.1 # Use 2.1 to enable using orbs and other features.
```

Comment:
What's the purpose of this config? It appears to be loading database schema.

Reply:
Whoops 😮 I need to kill this file. I was using this as a working reference while fixing all the broken schema changes from 2 to 2.1